spec/comment_spec.rb

```diff
@@ -0,0 +1,18 @@
+require 'rails_helper'
+
+RSpec.describe Comment, type: :model do
+  user = User.create(Name: 'Tom', Photo: 'https://unsplash.com/photos/F_-0BxGuVvo', Bio: 'Teacher from Mexico.')
+  post = Post.create(AuthorId: user.id, Title: 'Hello', Text: 'This is my first post')
+  subject { Comment.create(PostId: post.id, AuthorId: user.id, Text: 'Hi Tom!') }
+  before { subject.save }
+
+  it 'postId should be present' do
+    subject.PostId = nil
+    expect(subject).to_not be_valid
+  end
+
+  it 'author_id should be present' do
+    subject.AuthorId = nil
+    expect(subject).to_not be_valid
+  end
```

Comment:
- [ ] Great job working on the tests but I think you missed testing the method `update_comments_counter` in the Comment but since it is found in models so you should test it as shown in the requirements so kindly add test to the method ✨✅🔥

![image](https://user-images.githubusercontent.com/58553711/215096423-fbb8a9dd-0d87-4c7b-9e17-b994b5341eaf.png)